src/main/java/stringCalculator/domain/StringSplitter.java

```diff
@@ -0,0 +1,17 @@
+package stringCalculator.domain;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringSplitter {
+  private static final String DEFAULT_DELIMETER = ",|:";
+
+  public static String[] split(String input) {
+    Matcher matcher = Pattern.compile("//(.)\n(.*)").matcher(input);
```

Comment:
해당 메서드가 호출될때마다 패턴을 컴파일하도록 되어있네요!
패턴 컴파일은 시간이 많이 걸리는 프로세스이며 자원을 많이 사용하는것으로 알고 있습니다.
자바 doc에서도 한번 컴파일 후 재사용 할것을 권장하고 있습니다.
```
If a pattern is to be used multiple times, compiling it once and reusing
it will be more efficient than invoking this method each time
```
불필요한 객체 생성을 피하도록 캐싱하여 개선하면 좋을 것 같아요 :)

Reply:
아 그렇네요. split 메서드를 호출할 때마다 compile 작업이 일어나네요 ㅠㅠ step2 진행하기 전에 개선 시도해볼게요! 피드백 부탁드립니다 ㅎㅎ